### PR TITLE
Add info about the shared image registry

### PR DESCRIPTION
**Pull request by g-linville**

### CODE REVIEW

File: docs/docs/60-architecture/02-security-considerations.md
Comment (cjellick) [inline docs/docs/60-architecture/02-security-considerations.md:85]:
```suggestion
You can configure Acorn to use an external OCI registry, but the credentials are shared between all projects, so the security implications around image access are the same. Future releases will improve upon this by providing a framework for isolating registries.
```

**Review by cjellick — APPROVED** [review]
assuming you accept my edit, lgtm. and you dont have to wait for a second review

**Review by thedadams — APPROVED** [review]

**Pull request merged**
